libs/SmartStore/src/com/salesforce/androidsdk/smartstore/app/SmartStoreSDKManager.java

```diff
@@ -384,4 +389,31 @@ public void removeAllUserStores() {
                     UserAccountManager.getInstance().getCurrentUser().getCommunityId());
         }
     }
+
+    /**
+     * Setup global store using config found in res/raw/globalstore.json
+     */
+    public void setupGlobalStoreFromDefaultConfig() {
+        SmartStoreLogger.d(TAG, "Setting up global store using config sound in res/raw/globalstore.json");
+        setupStoreFromConfig(getGlobalSmartStore(), R.raw.globalstore);
+    }
+
+    /**
+     * Setup user store using config found in res/raw/userstore.json
+     */
+    public void setupUserStoreFromDefaultConfig() {
+        SmartStoreLogger.d(TAG, "Setting up user store using config sound in res/raw/userstore.json");
+        setupStoreFromConfig(getSmartStore(), R.raw.userstore);
+    }
+
+    /**
+     * Setup given store using config found in given json resource file
+     *
+     * @param store
+     * @param resourceId
+     */
+    private void setupStoreFromConfig(SmartStore store, int resourceId) {
```

Comment:
Should we make that third one public to allow configuring any stores from any resource file?

Reply:
I don't have a strong preference either way, but maybe we could expose this later? Developers might accidentally plug in other resources that aren't `JSON`.